CAIPs/caip-275.md

```diff
@@ -199,91 +202,82 @@ However, this configuration is not recommended as having multiple wallets genera
 Applications may also choose to directly integrate signer SDKs, providing a more streamlined signing flow.
 
 For example, here is how an application would integrate with the Login With Name login process:
 
-1. Install wagmi and [Login With Name WAGMI connector][] packages
-2. Import the loginWithName connector from the package
-3. Configure the wagmi config with the loginWithName connector.
-   This package provides ENS as a name resolver which can be configured to use mainnet or sepolia
-4. Wrap the app with WagmiProvider, pass it the configuration with the loginWithName connector
-5. Upon connection request, the connector will request application for user name. Application can show user an input for it if not obtained in other way.
+1. Install [Wagmi][] and [Login With Name WAGMI connector][] packages.
+2. Import the `loginWithName` connector from the package.
+3. Configure the [Wagmi][] config with the `loginWithName` connector.
+   This package provides ENS as a name resolver which can be configured to use `mainnet` or `sepolia`.
+4. Wrap the app with `WagmiProvider`, pass it the configuration with the loginWithName connector.
+5. Upon connection request, the connector will request application for user name. Application can show user an input for it, if not obtained in other way.
 6. Connector will now resolve user name into an address and authentication flows and trying to reach the wallet specified and its accounts.
-7. After signing in with the wallet. Application can use standard wagmi hooks like useAccount, useConnect etc.
+7. After signing in with the wallet. Application can use standard [Wagmi][] hooks like `useAccount`, `useConnect` etc.
 
 ## Name Resolver systems
 
 For the purposes of this document, we've detailed a flow based on ENS domains.
 But this standard is extensible to any name resolution system.
 For example, to use Solana domains, you could just replace the ENS name resolver component with a Solana name resolver component.
-The name resolver can also mix several sources, so resolving using several chains is only a matter of combining the specific name resolver for each chain and then combining them with the necessary logic
+The name resolver can also mix several sources, so resolving using several chains is only a matter of combining the specific name resolver for each chain and then combining them with the necessary logic.
 
 Reference implementation included in [Login With Name WAGMI connector][] include two name resolvers to showcase different implementations.
 
-- ENS, compatible with any chain that uses it and likely the one that will be the most common
-- Domain Wallets, an online web wallet system compatible with many chains and that already integrates a domain naming system to identify each wallet
-- An application level mixer name resolver, that combines the two previous ones
+- ENS, compatible with any chain that uses it and likely the one that will be the most common.
```

Comment:
```suggestion
- ENS, which has wide compatibility with many chains and integrated in many user-facing interfaces across web3.
```
maybe better not to explicit call out market-share, it makes competitors nervous